packages/playwright-core/bin/reinstall_msedge_beta_linux.sh

```diff
@@ -8,25 +8,22 @@ if [[ $(arch) == "aarch64" ]]; then
   exit 1
 fi
 
-is_user_root () { [ "${EUID:-$(id -u)}" -eq 0 ]; }
-if is_user_root; then
-  maybesudo=""
-else
-  maybesudo="sudo"
-fi
-
+# 1. make sure to remove old beta if any.
 if dpkg --get-selections | grep -q "^microsoft-edge-beta[[:space:]]*install$" >/dev/null; then
-  $maybesudo apt-get remove -y microsoft-edge-beta
+  apt-get remove -y microsoft-edge-beta
 fi
 
+# 2. Install curl to download Microsoft gpg key
 if ! command -v curl >/dev/null; then
-  $maybesudo apt-get install -y curl
+  apt-get update
```

Comment:
note: we have two apt-get updates here.
1. for installing cURL.
2. for updating the local package lists with the newly added msedge source list.